recipe/bld.bat

```diff
@@ -1,4 +1,11 @@
-move bin\* %LIBRARY_BIN%\
+REM Move only desired files as the downloaded package contains even Qt5 dlls
+move bin\cmake.exe %LIBRARY_BIN%\
+move bin\cmcldeps.exe %LIBRARY_BIN%\
+move bin\cpack.exe %LIBRARY_BIN%\
+move bin\ctest.exe %LIBRARY_BIN%\
+move bin\msvcp120.dll %LIBRARY_BIN%\
```

Comment:
Ok, it works without them. I checked Vista, win7 and win10 machines and they all provide these dlls. Don't know why they bundle them with cmake